x-pack/metricbeat/module/gcp/pubsub/_meta/data_topic.json

```
@@ -12,25 +12,112 @@
+                                66.24737266949234,
+                                79.4968472033908,
+                                95.39621664406897,
+                                114.47545997288276,
+                                137.3705519674593,
+                                164.84466236095116,
+                                197.8135948331414,
+                                237.37631379976966,
+                                284.8515765597236,
+                                341.82189187166824,
+                                410.18627024600187,
+                                492.22352429520225,
+                                590.6682291542427,
+                                708.8018749850912,
+                                850.5622499821095,
+                                1020.6746999785313,
+                                1224.8096399742376,
+                                1469.771567969085,
+                                1763.725881562902,
+                                2116.4710578754825,
+                                2539.7652694505787
+                            ],
+                            "counts": [
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                0,
+                                1,
+                                10,
+                                3,
+                                2
+                            ]
                         }
                     }
                 }
             }
+        },
+        "labels": {
+            "resource": {
+                "topic_id": "tommyers-topic"
```

Lets keep it as a generic topic name instead of this. 
```suggestion
                "topic_id": "test-topic"
```

---

thanks, fixed